pkg/bpf/mockbpf.go

```diff
@@ -1,17 +1,31 @@
-// Code generated by bpf2go; DO NOT EDIT.
-//go:build mips || mips64 || ppc64 || s390x
-
+//go:build !bpf
```

Comment:
The issue with this is that I'm pretty sure this will break the Go language server.

Currently, if I add a new map to the C code, run `make generate`, the language server will gladly autocomplete for me when I'm looking at `keplerObjects`, `keplerMaps` etc...
Given this is now the "default" behaviour, things like map renames etc.. will still compile just fine (even though it shouldn't) and you won't see the warnings in your editor 😢

Reply:
so yeah, this file would need to be manually updated for the structs to match... let me think on this a little more to see if there's anything else we can do here.